submissions/Maksym-Perepelytsia/html-css-popup/style.css

```diff
@@ -0,0 +1,204 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+    text-decoration: none;
+}
+
+body {
+    font-family: Tahoma, Helvetica, sans-serif;
+    font-size: 16px;
+    line-height: 1.5em;
+    background: #fff;
+}
+
+.wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100vh;
+}
+
+.header,
+.footer {
+    display: flex;
+    align-items: center;
+    background: #D3D3D3;
+    box-shadow: 0px 0px 3px 0px rgba(0, 0, 0, 0.5);
+}
+
+.header,
+.main,
+.footer{
+    padding: 15px;
+}
+
+.header {
+    justify-content: space-between;
+}
+
+.header__logo img,
+.popup-label img {
+    display:block;
+}
+
+.header__menu {
+    display: flex;
+}
+
+.header__menu__item .menu__link,
+.header__menu__item .popup-label{
+    padding:5px;
+    margin:5px;
+}
+
+.menu__link {
+    display: block;
+    color: #000;
+    position: relative;
+}
+
+.menu__link:after {
+    content: "";
+    position: absolute;
+    left:0;
+    bottom: 5px;
+    display: block;
+    opacity: 0;
+    width:100%;
+    height: 1px;
+    background: transparent;
+    transition: 0.5s;
+    transition-timing-function: cubic-bezier(.58,.3,.005,1);
+}
+
+.menu__link:hover:after {
+    opacity: 1;
+    background: linear-gradient(90deg, rgba(0,0,0,0) 0%, rgba(0,0,0,1) 50%, rgba(0,0,0,0) 100%);
+}   
+
+.popup-toggle {
+    border-radius: 50%;
+}
+
+#popup-checkbox,
+#more-cats-checkbox{
+    appearance: none;
+    position:absolute;
+}
+
+.popup-label{
+    display: flex;
+    cursor: pointer;
+}
+
+.popup-label:hover,
+#popup-checkbox:checked + .popup-label,
+#popup-checkbox:focus + .popup-label{
+    background-color: rgba(255,255,255,0.5);
+    box-shadow: 0px 0px 2px 0px rgba(0, 0, 0, 0.9);
+    border-radius: 50%;
+}
+
+#popup-checkbox:checked ~ .popup__wrapper {
+    display: block;
+}
+
+.popup__wrapper {
+    position: absolute;
+    right: 15px;
+    top: 70px;
+    display: none;
+    padding: 10px;
+    border: 1px solid rgba(0,0,0,.2);
+    border-radius: 5px;
+    background-color: #f5f5f5;
+}
+
+.popup__container {
+    padding:5px;
+    max-height: 260px;
+    overflow: auto;
+}
+
+.popup__menu {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 10px;
+    text-align: center;
+}
+
+.popup__menu__item a {
+    color: #000;
+    display: block;
+    padding: 5px;
+}
+
+.popup__menu__item a:hover {
+    border-radius:5px;
+    box-shadow: 0px 0px 2px 0px rgba(0, 0, 0, 0.8);
+}
+
+.popup__menu__item img {
```

Comment:
Please, avoid styling by tags [link](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)